File: 2520-count-the-digits-that-divide-a-number/2520-count-the-digits-that-divide-a-number.py
class Solution:
    def countDigits(self, num: int) -> int:
        original = num
        output=0
       
        while(num>0):
            z=num%10
            if(original%z==0 and z!=0):
                output+=1
            
            num=num//10
        
        return output
        

File: 2520-count-the-digits-that-divide-a-number/2520-count-the-digits-that-divide-a-number.java
class Solution {
    public int countDigits(int num) {
        int original = num;
        int z;
        int output=0;
        while(num>0){
            z=num%10;
            if(original%z==0){
                output++;
            }
            num=num/10;
        }
        return output;
    }
}